content/tracking/ua_ga4_migration.md

```diff
@@ -131,7 +131,7 @@ import { trackEnhancedEcommerceCheckoutStep } from '@snowplow/browser-plugin-sno
 trackEnhancedEcommerceCheckoutStep( {{dataLayer.ecommerce reference}}, checkoutOption);
 ```
 
-- Where `checkoutOption` is a key value pair object of available [Snowplow checkout options](https://github.com/snowplow/iglu-central/trenhancedecommerce/master/schemas/com.snowplowanalytics.snowplow.ecommerce/checkout_step/jsonschema), except `step` which is retrieved from the dataLayer directly.
+- Where `checkoutOption` is a key value pair object of available [Snowplow checkout options](https://github.com/snowplow/iglu-central/tree/master/schemas/com.snowplowanalytics.snowplow.ecommerce/checkout_step), except `step` which is retrieved from the dataLayer directly.
```

Comment:
```suggestion
- Where `checkoutOption` is a key value pair object of available [Snowplow checkout options](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow.ecommerce/checkout_step/jsonschema/1-0-0), except `step` which is retrieved from the dataLayer directly.
```

Reply:
I thought of linking to the folder so that moving forwards the user can choose the latest version of the schema and see all the available/unavailable fields. WDYT ?

Reply:
I guess for the user it's better if we tell them specifically which schema the tracker is using, but I can see the point that it is easier for us to maintain the docs if we don't have to update the schema versions on schema updates. Leaving it up to you which way you prefer, but would be better to at least link inside the `/jsonschema` folder so that the link goes to the list of schema files.

Reply:
Done thanks!